### PR TITLE
Add ShapeShift

### PR DESCRIPTION
**Pull request by mperklin**
I added an entry in the "CryptoCurrencies" section for ShapeShift digital asset exchange.

Thanks for maintaining this great repo!

--Michael

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
Please optimize your image. It must be under 2500 bytes.

**Review by stephengroat — DISMISSED** [review]

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -385,6 +385,14 @@ websites:
       software: Yes
       doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
 
+    - name: ShapeShift
+      url: https://shapeshift.io/
+      img: shapeshift.png
+      tfa: Yes
+      software: Yes
+      email: Yes
+      doc: https://shapeshift.zendesk.com/hc/en-us/articles/360001374159-I-Am-Having-Trouble-With-2FA
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml:394]: Change the URL to:
```suggestion
      doc: https://shapeshift.zendesk.com/hc/en-us/articles/360001374159
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Just as @mxxcon mentioned in https://github.com/2factorauth/twofactorauth/pull/3583#pullrequestreview-187020321, the logo needs to be less than 2500Kb. I compressed it down to 939 bytes.
Just replace shapeshift.png with this one:
shapeshift

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**